internal/core/api/workspace.josh.hcl

```diff
@@ -109,6 +109,7 @@ interface "workspace" {
     input "logs" "[]string" {}
 
     input "cursor" "*string" {}
+    input "filterStr" "*string" {}
```

Comment:
Seems like null/undefined and empty string have the same behavior, so we can probably just omit the `*`. We need some way to encode this conveniently for producing the right `filterStr?: string` in the typescript code, but like provide a hint or overlay or something or driven by `minLength` or something like that for Go.